daccs_client/node.py

```diff
@@ -11,16 +12,18 @@
 class DACCSNode:
     def __init__(self, nodename: str, jsondata: dict[str]) -> None:
         self._name = nodename
-        self._url = jsondata["url"]
+        self._description = jsondata["description"]
         self._date_added = dateutil.parser.isoparse(jsondata["date_added"])
         self._affiliation = jsondata["affiliation"]
-        self._icon_url = jsondata["icon_url"]
         self._location = jsondata["location"]
         self._contact = jsondata["contact"]
         self._last_updated = dateutil.parser.isoparse(jsondata["last_updated"])
         self._daccs_version = jsondata["version"]
         self._status = jsondata["status"]
 
+        for item in jsondata["links"]:
```

Comment:
We're making these private variables here. Is there a reason why we can't just store these as a dictionary instead of setting them as private attributes? It might be cleaner to organize internally and it won't matter to the user since it's private anyway

Reply:
I suppose that could be done. I just didn't think of that. Its not a bad idea.

Reply:
Just a suggestion :shrug: